libcodechecker/analyze/analysis_manager.py

```diff
@@ -566,6 +568,52 @@ def __create_timeout(analyzer_process):
                          (progress_checked_num.value, progress_actions.value,
                           action.analyzer_type, source_file_name))
 
+                # Remove the previously generated error file.
+                if os.path.exists(failed_dir):
+                    err_file = os.path.join(failed_dir, result_base + '.zip')
+                    if os.path.exists(err_file):
+                        os.remove(err_file)
+
+                if skip_handler:
+                    # We need to check the plist content because skipping
+                    # reports in headers can be done only this way.
+                    plist_parser.skip_report_from_plist(result_file,
+                                                        skip_handler)
+
+                try:
+                    source_files = plistlib.readPlist(result_file)['files']
+                    for sf in source_files:
+                        content_hash = None
```

Comment:
Is'n this code the same as one of the functions introduced in utils?

Reply:
Which function do you mean in the util.py?